compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -141,13 +141,9 @@ struct LoweringContext<'a, 'hir: 'a> {
     /// indicate whether or not we're in a place where new lifetimes will result
     /// in in-band lifetime definitions, such a function or an impl header,
     /// including implicit lifetimes from `impl_header_lifetime_elision`.
-    is_collecting_in_band_lifetimes: bool,
+    is_collecting_anonymous_lifetimes: bool,
```

Comment:
I renamed this to `anonymous`, since this handles both elided and in-band lifetimes, but we now only have the former. Open to calling this something like `is_collecting_elided_lifetimes` tho!